src/utils/wallet.js

```diff
@@ -689,7 +689,16 @@ class Wallet {
     async setupRecoveryMessageNewAccount(accountId, method, securityCode, fundingOptions, recoverySeedPhrase) {
         const { secretKey } = parseSeedPhrase(recoverySeedPhrase)
         const recoveryKeyPair = KeyPair.fromString(secretKey)
-        await this.validateSecurityCode(accountId, method, securityCode);
+        
+        try {
+            await this.validateSecurityCode(accountId, method, securityCode);
+        } catch (error) {
+            if (error.toString().includes('Error: Invalid code')) {
```

Comment:
Where is this error coming from? Latest near-api-js should have type field in errors.

Reply:
@vgrichina It's from CH, no `type`:
https://github.com/near/near-wallet/blob/master/src/utils/wallet.js#L665